manifests/resource/repository/docker/proxy.pp

```diff
@@ -54,6 +55,13 @@
   Enum['present', 'absent'] $ensure = 'present',
   Boolean $http_client_blocked = false,
   Boolean $http_client_auto_block = true,
+  Optional[Struct[{
+        type => Enum['username', 'ntlm '],
```

Comment:
`type => Enum['username', 'ntlm'],` i guess the space is not supposed to be there - else looks good for me now :)

Reply:
oupsy !